CHANGES.md

```diff
@@ -12,6 +12,7 @@
 * GAX has been upgraded to version 2.35.0
 * gRPC has been upgraded to version 1.58.0
 * Protocol Buffers has been upgraded to version 3.24.4
+* Enable read session caching by default for faster Spark planning
```

Comment:
Please change to `PR #1109:  Enable read session caching by default for faster Spark planning` and place after line 8 (to keep the PRs ordered)